src/overlays/effects/ovl_Effect_Ss_D_Fire/z_eff_ss_d_fire.h

```diff
@@ -11,8 +11,9 @@ typedef struct {
     /* 0x26 */ s16 scaleStep;
     /* 0x28 */ s16 alpha;
     /* 0x2A */ s16 fadeDelay;
-    /* 0x2C */ s32 life;
-} EffectSsDFireInitParams; // size = 0x30
+    /* 0x2C */ s16 life;
+    /* 0x30 */ s32 unk_2C;
```

Comment:
This should be renamed to match its offset.
```suggestion
    /* 0x30 */ s32 unk_30;
```